src/Sarif.UnitTests/FileDiffingTests.cs

```diff
@@ -0,0 +1,81 @@
+﻿// Copyright (c) Microsoft. All rights reserved. Licensed under the MIT        
+// license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Globalization;
+using System.IO;
+using Microsoft.CodeAnalysis.Sarif.Readers;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.CodeAnalysis.Sarif.UnitTests
+{
+    public class FileDiffingTests
```

Comment:
>FileDiffingTests [](start = 17, length = 16)

Move this file to `Sarif.TestUtilities`, and now, since it's a base class, I suggest `public abstract class FileDiffingTestBase`.